models/core/staging/core__stg_claims_procedure.sql

```diff
@@ -15,11 +15,7 @@ select
     claim_id as claim_id
   , patient_id as patient_id
   , claim_line_number as proc_seq_id
-  , coalesce(admission_date
-           , claim_start_date
-           , discharge_date
-           , claim_end_date
-    ) as procedure_date
+  , claim_line_start_date as procedure_date
```

Comment:
Have we checked with the tuva community (or newer versions of this model) to see if this is correct? Also, wondering if we should just add this as the first item in the coalesce, or if it's better to have no date given the other dates being coalesced may produce incorrect results?

Reply:
yeah, that was my first instinct, but I am worried about correctness than missingness. And yes, it is still like this in Tuva's main branch. I reached out to Thu at Tuva for her thoughts on this